Media/CollectionStrategyInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace L91\Sulu\Bundle\FormBundle\Media;
+
+/**
+ * Interface for the collection strategy.
+ */
+interface CollectionStrategyInterface
+{
+    /**
+     * Get collection id.
+     *
+     * @param int $formId
+     * @param string $formTitle
+     * @param string $type
+     * @param string $typeId
+     * @param string $title
+     * @param string $locale
+     *
+     * @return array|int
+     */
+    public function getCollectionId(
```

Comment:
currently I would let it be as it is because in the future the media collection and co is not needed to create the Dynamic.php